src/Find.js

```diff
@@ -1,60 +1,46 @@
 import React, { Component } from 'react'
-import CardFront from './CardFront'
-// import onePlayer from './SinglePlayerData.json'
-// import allPlayers from './WholeTeamData.json'
+import Select from 'react-select'
 import './Find.css'
 
 class Find extends Component {
     constructor() {
         super()
         this.state = {
-            playerData: {}
+            chosenPlayer: {},
+            playerID: null
         }
     }
 
     handleChange = (event) => {
-        this.setState({ playerName: event.target.value })
+        this.props.showFrontOrBack('front')
+        this.setState({ playerID: event.value })
+        const chosenPlayer = this.getSinglePlayerData(event)
+        this.props.setCurrentPlayer(chosenPlayer)
```

Comment:
When you choose a player from the dropdown list:
- set App to show the front of the card
- put that player's ID into state
- get that player's data object
- set that player's data object into App state

I might be able to do away with putting the playerID into state here, but since it's working right now I'm going to focus on other things.